frontend-react/src/components/Admin/EditSenderSettings.tsx

```diff
@@ -119,12 +119,10 @@ const EditSenderSettingsForm: React.FC<EditSenderSettingsFormProps> = ({
 
     const ShowCompareConfirm = async () => {
         try {
-            const { prohibited, errorMsg } = isProhibitedName(
```

Comment:
Some of the logic here is duplicated [in NewSetting.tsx](https://github.com/CDCgov/prime-reportstream/pull/8760/files#diff-ba4adccf9523006412c5595106a2679a0f83f9537a90816cf1f6c15929951bd7R40-R43).  I'm okay with this for now because my feeling is that we'll want to spend more time consolidating these components in the long run; the flow should be similar for adding and editing service settings in the future.